.github/workflows/main.yml

```diff
@@ -98,8 +98,6 @@ jobs:
             osName: windows
           - os: macOS-latest
             osName: mac
-          - jdk: 11
-            deploy: false
```

Comment:
You should revert this, and instead set `deploy: true` on jdk 17.

Reply:
By the way, what will happen if we just set deploy: true on jdk 17 instead of removing it from "include". Will it deploy with all OS?

Reply:
The include just adds additional flag for each runner image.

Reply:
I am going to explicitly evaluate the jdk11 to `deploy: false` and jdk17 to `deploy: true` and update the comments, I think it will be much better if we specify flags explicitly, do we agree on these changes ?

Reply:
>and jdk17 to deploy: true

I wonder if it will also deploy with mac and windows as well if you set it to `true`? I need to take a look at the docs to see how `include` works.

Reply:
And I will also create deploy: false explicitly for other systems (Mac and windows) to avoid confusion.

Reply:
>And I will also create deploy: false explicitly for other systems (Mac and windows) to avoid confusion.

[mac:deploy=false, jdk17: deploy=true] 
[windows:deploy=false, jdk17: deploy=true] 

Will the `deploy` defined for jdk overwrite the one in os?

Reply:
It shouldn't, the old code currently in the master defines only jdk 8 with true and Ubuntu with true, so the runner which runs Ubuntu jdk8 will deploy, deploy is always false unless explicitly defined by the runner as true.

Reply:
I will see.

Reply:
Long story short, we shouldn't set `deploy: true` on a jdk runner because it will override the previously defined flag.

Reply:
I see.